front-end/src/app/transactions/transaction-group-h/transaction-group-h.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { TransactionTypeBaseComponent } from 'app/shared/components/transaction-type-base/transaction-type-base.component';
+import { LabelUtils, PrimeOptions } from 'app/shared/utils/label.utils';
+import { ContactTypeLabels, ContactTypes } from '../../shared/models/contact.model';
+
+@Component({
+  selector: 'app-transaction-group-h',
+  templateUrl: './transaction-group-h.component.html',
+})
+export class TransactionGroupHComponent extends TransactionTypeBaseComponent implements OnInit, OnDestroy {
+  formProperties: string[] = [
+    'entity_type',
+    'payee_organization_name',
+    'payee_last_name',
+    'payee_first_name',
+    'payee_middle_name',
+    'payee_prefix',
+    'payee_suffix',
+    'payee_street_1',
+    'payee_street_2',
+    'payee_city',
+    'payee_state',
+    'payee_zip',
+    'expenditure_date',
+    'expenditure_amount',
+    'aggregate_amount',
+    'expenditure_purpose_descrip',
+    'memo_code',
+    'memo_text_input',
+    'category_code',
+  ];
+  override contactTypeOptions: PrimeOptions = LabelUtils.getPrimeOptions(ContactTypeLabels).filter((option) =>
```

Comment:
since @Elaine-Krauss-TCG added the `values` arg to getPrimeOptions, this could be replaced with 
```
override contactTypeOptions: PrimeOptions = LabelUtils.getPrimeOptions(ContactTypeLabels, [ContactTypes.INDIVIDUAL, ContactTypes.ORGANIZATION, ContactTypes.COMMITTEE])
```

Reply:
Good call. This change has been made for all the group templates and in the contact input form.